src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static get alive() {
```

Comment:
I don't think this should be a getter, that's just the method we use inside our class